Assets/Scripts/Minigames/CowardMiniGame/WrenchBehavior.cs

```diff
@@ -72,32 +75,39 @@ IEnumerator Swinging()
     {
         //print("swing");
         GetComponent<Collider>().enabled = true;
-        _animate.SetBool("isSwinging", true);
+        _animate.SetBool("_isSwinging", true);
         //_swing = true;
-        yield return new WaitForSeconds(0.1f);
+        yield return new WaitForSeconds(1f);
         //_swing = false;
         GetComponent<Collider>().enabled = false;
-        _animate.SetBool("isSwinging", false);
+        _animate.SetBool("_isSwinging", false);
+    }
+    IEnumerator SystematicShutDown()
+    {
+        yield return new WaitForSeconds(1.1f);
+        gameObject.SetActive(false);
     }
-
     /// <summary>
     /// This function is invoked in SparkInteractBehavior whenever a spark is interacted
     /// with. It keeps track of the number of sparks that have been smacked and ends the game.
     /// </summary>
     private void SparkSmacked()
     {
-        _sparkSmacked++;
-        _smackedText.text = _sparkSmacked.ToString();
-        StartCoroutine(Swinging());
-        if (_sparkSmacked >= 5)
+        if (_isEquipped == true)
         {
-            _smackedText.color = Color.green;
-            _sparksMode.SetActive(false);
-            gameObject.SetActive(false);
+            _sparkSmacked++;
+            _smackedText.text = _sparkSmacked.ToString();
+            StartCoroutine(Swinging());
+            if (_sparkSmacked >= _maxSpark)
+            {
+                _smackedText.color = Color.green;
+                _sparksMode.SetActive(false);
+                StartCoroutine(SystematicShutDown());
 
-            //game ends here?
-            _minigameEndEvent.TriggerEvent(NpcEventTags.Coward);
-            print("game end");
+                //game ends here?
+                _minigameEndEvent.TriggerEvent(NpcEventTags.Coward);
+                print("game end");
```

Comment:
Remove debug statements at this point, we're nearing GM and don't need these in the final code